documentation/description.md

```diff
@@ -1225,6 +1227,57 @@ disruption ongoing. We can tune this by updating the `spec.maxPodUnavailable` pa
 
 > **IMPORTANT:** it is recommended to not touch this parameter unless you know what you are doing.
 
+### Cross Ip Management
+
+#### Global mecanism
+
+Cassandra works on IPs and not on hostname, so in a case where two cassandra cross their Ips, the cassandra node will not being able to run properly and will loop on the following error :
```

Comment:
will not be able

Reply:
the Cassandra Node -> no Cassandra node. (Let's keep Cassandra titled also)